fe/src/main/java/org/apache/doris/master/ReportHandler.java

```diff
@@ -247,6 +249,13 @@ private static void tabletReport(long backendId, Map<Long, TTablet> backendTable
         LOG.info("backend[{}] reports {} tablet(s). report version: {}",
                  backendId, backendTablets.size(), backendReportVersion);
 
+        final SystemInfoService currentSystemInfo = Catalog.getCurrentSystemInfo();
+        Backend reportBackend = currentSystemInfo.getBackend(backendId);
+        if (reportBackend != null) {
+            BackendStatus backendStatus = reportBackend.getBackendStatus();
+            backendStatus.lastTabletReportTime = TimeUtils.longToTimeString(start);
```

Comment:
Better to create a method called `backend.updateStatus()`  to set these infos. 
Maybe we will add more info later.

Reply:
if we need add more info, we can just add a new Field in `BackendStatus` and set the field's value, and new info will be shown in `show backends`, why we need to set the value and update the value?I think we can just set the value, and the new value will be shown when we use `show backends`?

Reply:
It ok, not a big deal.
Oh, I suggest to set this value right after the report in done, in case some other error happens during the report process.

Reply:
The status mainly reflect BE's status, if some error happends, it should be FE's problem?Our mainly purpose is to monitoring BE's status, so maybe we can leave it to make sure that BE's status is OK?